excel_import_export/models/common.py

```diff
@@ -268,13 +268,12 @@ def _get_cell_value(cell, field_type=False):
     datemode = 0  # From book.datemode, but we fix it for simplicity
     if field_type in ['date', 'datetime']:
         ctype = xlrd.sheet.ctype_text.get(cell.ctype, 'unknown type')
-        if ctype == 'number':
+        if ctype in ('xldate', 'number'):
+            is_datetime = cell.value % 1 != 0.0
```

Comment:
I have a question. This kind of check is safe? Cannot fail? Isn't it better to check `field_type == 'datetime'`? Just asking.

Reply:
I am not too certain. But as I saw in Libre office, for the field what is Date, i.e., 2010-10-10, when chagned cell type to text, number or general, it is changed to 40461. So, I just keep that on the safe side (some how).
WDYT?

Reply:
Also, comparing to 'number' used to work on my older versions (back in v8).

Reply:
ok